__tests__/integration.test.js

```diff
@@ -94,3 +94,45 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("200: responds with an articles array of objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((property) => {
+          expect(typeof property.article_id).toBe("number");
+          expect(typeof property.title).toBe("string");
+          expect(typeof property.topic).toBe("string");
+          expect(typeof property.author).toBe("string");
+          expect(typeof property.created_at).toBe("string");
+          expect(typeof property.votes).toBe("number");
+          expect(typeof property.article_img_url).toBe("string");
+          expect(typeof property.comment_count).toBe("string");
+        });
+      });
+  });
+  test("200: should respond with an array of articles without a body property present on any of the objects", () => {
```

Comment:
This is also ok to combine with the above test in your forEach. I would refactor the type of to be objectContaining and then an extra toHaveProperty on the body to be false. Much more concise as the test above doesn't prove the body isn't there, but could do.